Documentation/custom_effects.md

```diff
@@ -48,6 +48,104 @@ These are some tips for writing or converting effects for use with MonoGame.
 * The effect compiler is aggressive about removing unused paramters, be sure the parameters you are setting are actually used.
 * If you think you've found a bug porting a shader [please let us know](https://github.com/mono/MonoGame/issues).
 
+### Summary of some work arounds in the process of transferring effects files across to MonoGame from XNA
+
+#### Aligning your version of the content processor and the runtime
+For shader .fx files, the version of the of the content processor you are using to compile your xnb files (ie. **MonoGameContentProcessors.dll**) **must be aligned** with the version of the runtime you are running your game against (ie. **MonoGame.Framework.dll**)
+
+Otherwise you may get **"Wrong MGFX file version!"** raised by ReadEffect() in Effect.cs when you do a **content.Load<Effect>**
+[because the version written in to the .xnb file is not a match with the value of MGFXVersion 
+that has been put in the run time].
+
+So if you have this error, the first thing to check is that your content processor is the same up-to-dateness as the version of the runtime you are running your game against.
+
+
+#### Shader model compilation:
+(a) For **OpenGL** versions of MonoGame (eg. Mac, Linus, WindowsGL) the shader model must be **SM 3.0 or lower**
```

Comment:
Please stop using `(a)`, `(b)`, `(c)` to itemize things.  There are good ways to do this [via markdown (read this)](https://guides.github.com/features/mastering-markdown/).